src/content/docs/alerts-applied-intelligence/new-relic-alerts/get-started/choose-your-aggregation-method.mdx

```diff
@@ -0,0 +1,96 @@
+
+A data point with a 12:00pm timestamp arrives. An aggregation window opens. At some point, a 12:03pm data point arrives. Event flow closes the window, excluding the 12:03 data point, and evaluates that closed window against your thresholds.
+
+The event flow aggregation window will continue collecting data points until that later timestamp. The later timestamps are what moves the system forward, not the data points themselves. Event flow will wait as long as necessary for the next data point later than your delay setting to arrive, before aggregating the data.
+
+Event flow works best for data that arrives frequently and consistently.
+
+## Event flow use cases [#event-flow-use-cases]
+
+Here are some typical event flow use cases:
+
+* APM agent data.
+* Infrastructure agent data.
+* Any data coming from a 3rd party that comes in frequently and reliably.
+* Most AWS Cloudwatch metrics coming from the AWS Metric Stream (NOT polling). The main exception is that some AWS Cloudwatch data is very infrequent (like S3 volume data) regardless of whether it's streaming or polling and, in that case, you'd use **Event timer**.
+
+## When to use event timer [#event-timer]
+
+When data arrives sporadically, inconsistently, and out of order, we recommend using event timer aggregation.
+
+![A screenshot showing where you can find the NRQL query builder button in New Relic One.](./images/find-builder.png 'A screenshot showing where you can find the NRQL query builder button in New Relic One.')
+
+Event timer aggregation is based on a timer that counts down when a data point arrives. The timer resets every time a new data point arrives. If the timer counts down before a new data point arrives, event timer aggregates all of the data points received during that time.
+
+Event timer is best for alerting on events that happen sporadically and with large gaps of time.
+
+## How event timer works [#event-flow-detail]
+
+Errors are a type of event that happens sporadically, unpredictably, and often with large gaps of time.
+
+For example, you might have a condition with a query that returns a count of errors. Many minutes may go by with errors at all, and then suddenly 5 errors arrive within a minute.
+
+In this example, event timer would do nothing until the first of the 5 errors arrive. Then it would start the timer, resetting it each time a new error arrives. If the timer countdown reaches 0 without a new error, event timer aggregates the data, and evaluates it against your threshold.
+
+## Event timer use cases [#event-timer-use-cases]
+
+Here are some typical event timer use cases:
+
+* New Relic usage data.
+* Cloud integration data that is being polled (such as with GCP, Azure, or AWS polling methods).
+* Queries that deliver sparse or infrequent data, such as error counts.
+
+## Cadence [#cadence]
+
+Cadence is our original aggregation method. We recommend that you use one of our other aggregation methods instead.
```

Comment:
Should we at least explain what it is? It seems weird we have it as an option, but then tell people to just not use it.

Reply:
We'd really rather people don't use it. The only reason we haven't simply removed it is that we have customers currently using it.

Reply:
I've added a suggestion about what to do if you're using cadence, based on the data you're alerting on.